controllers/user.js

```diff
@@ -555,6 +562,23 @@ async function getBreachStats(req, res) {
 
 
 function logout(req, res) {
+  // Growth Experiment
+  if (EXPERIMENTS_ENABLED && req.session.experimentFlags) {
+    // Persist experimentBranch across session reset
+    const excludeFromExperiment = req.session.experimentFlags.excludeFromExperiment;
+    const experimentBranch = req.session.experimentFlags.experimentBranch;
+    req.session.reset();
+    // Reset session vars after sign out event
+    req.session.experimentFlags = {
+      experimentBranch,
+      excludeFromExperiment,
+    };
```

Comment:
There's some clever de-structuring and re-structuring way to do this with less code, but I'm not clever.

Reply:
Are there other experiment variables in `req.session.experimentFlags` that you want to reset or could this maybe just be something like...
```
if (EXPERIMENTS_ENABLED && req.session.experimentFlags) {
   const sessionExperimentFlags = req.session.experimentFlags
   req.session.reset();
   req.session.experimentFlags = sessionExperimentFlags
};

Reply:
@lesleyjanenorton Ohh. That's exactly it. There aren't any. We can/should just save the entire object. 🤝